specification/storagecache/resource-manager/Microsoft.StorageCache/stable/2020-03-01/storagecache.json

```diff
@@ -0,0 +1,1776 @@
+      "properties": {
+        "nextLink": {
+          "description": "URL to get the next set of operation list results if there are any.",
+          "type": "string"
+        },
+        "value": {
+          "description": "List of Resource Provider operations supported by the Microsoft.StorageCache resource provider.",
+          "items": {
+            "$ref": "#/definitions/ApiOperation"
+          },
+          "type": "array"
+        }
+      },
+      "type": "object"
+    },
+    "CloudError": {
+      "x-ms-external": true,
+      "description": "An error response.",
+      "properties": {
+        "error": {
+          "description": "The body of the error.",
+          "$ref": "#/definitions/CloudErrorBody"
+        }
+      },
+      "type": "object"
+    },
+    "CloudErrorBody": {
+      "description": "An error response.",
+      "properties": {
+        "code": {
+          "description": "An identifier for the error. Codes are invariant and are intended to be consumed programmatically.",
+          "type": "string"
+        },
+        "details": {
+          "description": "A list of additional details about the error.",
+          "items": {
+            "$ref": "#/definitions/CloudErrorBody"
+          },
+          "type": "array"
+        },
+        "message": {
+          "description": "A message describing the error, intended to be suitable for display in a user interface.",
+          "type": "string"
+        },
+        "target": {
+          "description": "The target of the particular error. For example, the name of the property in error.",
+          "type": "string"
+        }
+      },
+      "type": "object"
+    },
+    "Cache": {
+      "description": "A Cache instance. Follows Azure Resource Manager standards: https://github.com/Azure/azure-resource-manager-rpc/blob/master/v1.0/resource-api-reference.md",
+      "properties": {
+        "tags": {
+          "description": "ARM tags as name/value pairs.",
+          "type": "object"
+        },
+        "id": {
+          "description": "Resource ID of the Cache.",
+          "readOnly": true,
+          "$ref": "#/definitions/URLString"
+        },
+        "location": {
+          "description": "Region name string.",
+          "type": "string"
+        },
+        "name": {
+          "description": "Name of Cache.",
+          "readOnly": true,
+          "$ref": "#/definitions/ResourceName"
+        },
+        "type": {
+          "description": "Type of the Cache; Microsoft.StorageCache/Cache",
+          "readOnly": true,
+          "type": "string"
+        },
+        "identity": {
+          "$ref": "#/definitions/CacheIdentity",
+          "description": "The identity of the cache, if configured."
+        },
+        "properties": {
+          "x-ms-client-flatten": true,
+          "description": "Properties of the Cache.",
+          "properties": {
+            "cacheSizeGB": {
```

Comment:
is there boundary of the size?

Reply:
The cacheSizeGB can't be arbitrary value and the value should use one of values defined in Skus.

Reply:
then type is enum?

Reply:
The reason we didn't use enum here is that if we add more cache size entries to our skus then we need to update the swagger each time. Our RP validates the size and would throw an error if size is not in one of our provided skus. Users usually need to find out what cache skus we offer first with get skus API, then use one of values in the skus to create a cache.